pypesto/sample/parallel_tempering.py

```diff
@@ -71,6 +78,8 @@ def default_options(cls) -> Dict:
             "exponent": 4,
             "temper_log_posterior": False,
             "show_progress": None,
+            "beta_init": "exponential_decay",
```

Comment:
Is one of `beta_decay` or `exponential_decay` better in general? Since `beta_decay` is better for evidence, should we change the default to `beta_decay`?

Reply:
If you use the adaptive parallel tempering then you should use `exponential_decay`, that seems to be what Vousden et al. 2016. proposes. I do not think anybody will use the non-adaptive version, if not for evidence estimation so we can switch the default here.

Reply:
Sounds good :+1: The choice of decay method should be communicated to the user somehow, e.g. docs or logs. Probably easiest to do some `logging.info` message that prints the options used, either always or only when the user doesn't specify the decay method